package.json

```diff
@@ -12,29 +12,35 @@
   },
   "scripts": {
     "build": "gulp",
-    "create-push-notifications-service-user": "node dist/tools/createPushNotificationsServiceUser.js",
-    "create_db": "sequelize db:create",
-    "create_missing_users": "node dist/tools/create_missing_users.js",
-    "delete_db": "sequelize db:drop",
-    "jenkins": "cross-env NODE_ENV=test yarn delete_db; cross-env NODE_ENV=test yarn create_db && cross-env NODE_ENV=test yarn migrate && cross-env NODE_ENV=test yarn test",
+    "create-push-notifications-service-user": "babel-node tools/createPushNotificationsServiceUser.js",
+    "jenkins": "cross-env NODE_ENV=test yarn mongo:delete-db && cross-env NODE_ENV=test yarn test",
     "lint": "esw -c .eslintrc *.js server config --color",
     "lint:fix": "yarn lint --fix",
     "lint:watch": "yarn lint --watch",
-    "migrate": "sequelize db:migrate",
-    "migrate:undo": "sequelize db:migrate:undo",
+    "mongo:create-missing-users": "babel-node tools/createMissingUsers.js",
+    "mongo:delete-db": "babel-node tools/deleteDb.js",
+    "mongo:fail-on-db-exists": "babel-node tools/failOnDbExists.js",
+    "mongo:migrate:up": "db-migrate --config migrate.config.json up",
+    "mongo:migrate:down": "db-migrate --config migrate.config.json down",
     "report-coverage": "coveralls < ./coverage/lcov.info",
     "serve": "node ./dist/index.js",
     "snyk-protect": "snyk protect",
+    "sql:create-missing-users": "babel-node sql/tools/create_missing_users.js",
+    "sql:create-db": "sequelize db:create",
+    "sql:delete-db": "sequelize db:drop",
+    "sql:migrate": "sequelize db:migrate",
+    "sql:migrate:undo": "sequelize db:migrate:undo",
+    "sql:update-uuid": "babel-node sql/tools/addUUID.js",
+    "sql:to-mongo": "babel-node sql/tools/toMongo.js",
     "start": "gulp serve",
     "start:debug": "cross-env DEBUG=amida-notification-microservice:* yarn start",
-    "test": "cross-env NODE_ENV=test mocha --opts .mocharc",
+    "test": "cross-env NODE_ENV=test yarn mongo:fail-on-db-exists && cross-env NODE_ENV=test mocha --opts .mocharc",
```

Comment:
Conclusion of our discussion:

Make this script do something like this
```json
"test": "cross-env NODE_ENV=test yarn mongo:delete-db && cross-env NODE_ENV=test mocha --opts .mocharc",
```

And modify `tools/deleteDb.js` such that it
1. If `NODE_ENV==='test'`, just drops your DB.
2. If `NODE_ENV!=='test'`, it warns the user that NODE_ENV isn't test, and that it's about to drop the database named X, and asks the user to type y or yes or something to confirm.